tests/Browser/Select/StyledCommonTest.php

```diff
@@ -291,6 +291,49 @@ public function can_select_multiple_with_same_label(): void
             ->waitForText('baz');
     }
 
+    /** @test */
+    public function can_select_with_disabled_option(): void
```

Comment:
Which of these tests validates that it is not possible to select a disabled option?

Reply:
can_select_with_disabled_option